src/composite.jl

```diff
@@ -0,0 +1,54 @@
+"""
+   Composite <: Transform
+
+A `Composite` transform is a composition of `Transform`s, currently limited to `OneToOne()`
+cardinality. It can be fit and applied in a single step.
+
+The transforms in `Composite([t1, t2, t3])` are applied in `t1`, `t2`, `t3` order, where
+the output of `t1` is the input to `t2` etc. When using `∘` to create transforms, the order
+is `t3 ∘ t2 ∘ t1`, as in function composition.
+
+```jldoctest composite
+julia> id = IdentityScaling();
+
+julia> power = Power(2.0);
+
+julia> id ∘ power == Composite([power, id])
+true
+```
+"""
+struct Composite <: Transform
+    transforms::Vector{<:Transform}
```

Comment:
```suggestion
    transforms::Tuple{Vararg{Transform}}
```

more memory efficient, and allows for an external constructor like
```julia
Composite(transforms::Transform...) = Composite(transforms)
```